### ISSUE TITLE
nbconvert broken in python 2.7

### ISSUE THREAD

**Issue by ianhbell** [jupyter/nbconvert#278]
In python 2.7 64-bit, on windows, nbconvert is broken.  For python 3.5 (and I suspect 3.x), it works just fine.

I made a simple dummy notebook, with the contents:
![capture](https://cloud.githubusercontent.com/assets/11966765/14124725/5ec1fa34-f5c4-11e5-9e18-1327ed282dca.PNG)

and tried to convert to pdf with 
```
[py27bare] C:\REDACTEDPATH>jupyter nbconvert --to pdf Untitled1.ipynb
[NbConvertApp] Converting notebook Untitled1.ipynb to pdf
[NbConvertApp] Writing 13747 bytes to notebook.tex
[NbConvertApp] Building PDF
[NbConvertApp] Running pdflatex 3 times: ['pdflatex', 'notebook.tex']
Traceback (most recent call last):
  File "D:\Anaconda\envs\py27bare\Scripts\jupyter-nbconvert-script.py", line 5, in <module>
    main()
  File "D:\Anaconda\envs\py27bare\lib\site-packages\jupyter_core\application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "D:\Anaconda\envs\py27bare\lib\site-packages\traitlets\config\application.py", line 596, in launch_instance
    app.start()
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\nbconvertapp.py", line 289, in start
    self.convert_notebooks()
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\nbconvertapp.py", line 412, in convert_notebooks
    self.convert_single_notebook(notebook_filename)
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\nbconvertapp.py", line 383, in convert_single_notebook
    output, resources = self.export_single_notebook(notebook_filename, resources)
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\nbconvertapp.py", line 335, in export_single_notebook
    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\exporters\exporter.py", line 165, in from_filename
    return self.from_notebook_node(nbformat.read(f, as_version=4), resources=resources, **kw)
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\exporters\pdf.py", line 144, in from_notebook_node
    rc = self.run_latex(tex_file)
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\exporters\pdf.py", line 107, in run_latex
    self.latex_count, log_error)
  File "D:\Anaconda\envs\py27bare\lib\site-packages\nbconvert\exporters\pdf.py", line 87, in run_command
    p = subprocess.Popen(command, stdout=stdout, stdin=null, shell=shell, env=env)
  File "D:\Anaconda\envs\py27bare\lib\subprocess.py", line 710, in __init__
    errread, errwrite)
  File "D:\Anaconda\envs\py27bare\lib\subprocess.py", line 958, in _execute_child
    startupinfo)
TypeError: environment can only contain strings
```
My suspicion is a bug in the shell encoding logic in pdf.py, but I can't make out any more than that.

My conda enviornment:
```
[py27bare] C:\REDACTEDPATH>conda list
# packages in environment at D:\Anaconda\envs\py27bare:
#
backports-abc             0.4                       <pip>
backports.ssl-match-hostname 3.4.0.2                   <pip>
backports_abc             0.4                      py27_0    <unknown>
decorator                 4.0.9                    py27_0    defaults
ipykernel                 4.3.1                    py27_0    defaults
ipython                   4.1.2                    py27_1    defaults
ipython-genutils          0.1.0                     <pip>
ipython_genutils          0.1.0                    py27_0    <unknown>
jinja2                    2.8                      py27_0    <unknown>
jsonschema                2.4.0                    py27_0    <unknown>
jupyter-client            4.2.2                     <pip>
jupyter-core              4.1.0                     <pip>
jupyter_client            4.2.2                    py27_0    defaults
jupyter_core              4.1.0                    py27_0    defaults
markupsafe                0.23                     py27_1    defaults
mistune                   0.7.2                    py27_0    defaults
nbconvert                 4.1.0                    py27_0    <unknown>
nbformat                  4.0.1                    py27_0    <unknown>
notebook                  4.1.0                    py27_2    defaults
path.py                   8.1.2                    py27_1    <unknown>
pickleshare               0.5                      py27_0    <unknown>
pip                       8.1.1                    py27_1    defaults
pygments                  2.1.1                    py27_0    defaults
pyreadline                2.1                      py27_0    <unknown>
python                    2.7.11                        4    defaults
pyzmq                     15.2.0                   py27_0    <unknown>
setuptools                20.3                     py27_0    defaults
simplegeneric             0.8.1                    py27_0    <unknown>
singledispatch            3.4.0.3                  py27_0    <unknown>
six                       1.10.0                   py27_0    <unknown>
ssl_match_hostname        3.4.0.2                  py27_0    <unknown>
tornado                   4.3                      py27_0    <unknown>
traitlets                 4.2.1                    py27_0    defaults
vs2008_runtime            9.00.30729.1                  0    defaults
wheel                     0.29.0                   py27_0    defaults
```

**Comment by takluyver** [comment]
Already fixed by #165, but we need to do a new release to get the fix out.

**Comment by ianhbell** [comment]
Thanks for the info.. Any prognosis on a new conda package?
